notification-portlet-webapp/src/main/webapp/scripts/jquery.notice.js

```diff
@@ -93,8 +94,14 @@ if (!upnotice.init) {
               var actionElement = actionTemplate.clone();
               actionElement.removeClass('action-template');
               actionElement.toggleClass('hidden');
-              actionElement.find('a').attr('href', actionUrl).html(action.label + " ");
+              if( action.id == 'StateChangeAction') {
```

Comment:
I suggest adding a comment explaining that the StateChangeAction decorates the link rather than being listed to the side as is done with the other actions.